lib/internal/source_map/source_map_cache.js

```diff
@@ -108,6 +119,31 @@ function maybeCacheSourceMap(filename, content, cjsModuleInstance) {
   }
 }
 
+function maybeCacheGeneratedSourceMap(content) {
+  const sourceMapsEnabled = getSourceMapsEnabled();
+  if (!(process.env.NODE_V8_COVERAGE || sourceMapsEnabled)) return;
+
+  const matchSourceURL = StringPrototypeMatch(
+    content,
+    /\/[*/]#\s+sourceURL=(?<sourceURL>[^\s]+)/
+  );
```

Comment:
https://github.com/nodejs/node/blob/358008f4738e78fd34aa5428ef2d86bbf093beb9/doc/contributing/primordials.md#L566-L568
```suggestion
  const matchSourceURL = RegExpPrototypeExec(
    /\/[*/]#\s+sourceURL=(?<sourceURL>[^\s]+)/,
    content
  );
```

Reply:
Thank you for the suggestion! Fixed.